po/de.po

```diff
@@ -108,34 +112,30 @@ msgid "Enter key sends"
 msgstr "Eingabetaste sendet"
 
 #: ../qml/phoneui/ui/settings/AdvancedPage.qml:87
-#, fuzzy
 msgid "Activate Debuglog"
-msgstr "Diagnoseprotokoll übermitteln"
+msgstr "Diagnoseprotokoll aktivieren"
 
 #: ../qml/phoneui/ui/settings/LinkedDevicesPage.qml:13
 #: ../qml/phoneui/ui/settings/LinkedDevicesPage.qml:17
 #: ../qml/phoneui/ui/settings/AddLinkedDevicesPage.qml:10
 msgid "Link a Device"
-msgstr ""
+msgstr "Gerät verknüpfen"
```

Comment:
I'd rather use "Geraet hinzufuegen" or "Geraet verbinden". "Verknuepfen" is rather seldom used in this context, eve though "link" would translate to "verknuepfen".

Reply:
Sounds good. "Gerät verbinden" would be fine with me.